vlog4j-client/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.semanticweb.vlog4j</groupId>
+		<artifactId>vlog4j-parent</artifactId>
+		<version>0.5.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>vlog4j-client</artifactId>
+	<packaging>jar</packaging>
+	<name>VLog4j Client</name>
+	<description>Stand-alone VLog4j application</description>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>vlog4j-core</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>vlog4j-owlapi</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>vlog4j-rdf</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>vlog4j-parser</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
```

Comment:
Now it only uses `vlog4j-core`, `vlog4j-parser`, and `vlog4j-examples` (to print query answers). In the future it should use the rest of the modules. I'll remove the unused dependencies, and add them when necessary.

Reply:
Since vlog4j-examples has dependencies on almost every module, we can create a lightweight vlog4j-utils module in another issue, and move there the methods for printing query answers.

create vlog4j-utils module #124